cmd/sriov-network-config-daemon/start.go

```diff
@@ -178,7 +199,10 @@ func runStartCmd(cmd *cobra.Command, args []string) {
 		startOpts.nodeName,
 		snclient,
 		kubeclient,
-		mcclient,
+		utils.OpenshiftContext{
```

Comment:
i think it would be better if daemon will store a pointer to OpenshiftContext, that way its nil when cluster is not openshift. WDYT ?